CHANGELOG.md

```diff
@@ -27,24 +27,26 @@ Additional settings for console behavior have been added:
 - Hide Code Input – When enabled, only the execution output is shown in the console.
 - Clear Cells on Execute – When enabled, only the last cell is displayed.
 
+<img alt="The option to change console prompt position is available under the three dots button in the console toolbar, which is the last toolbar button" src="https://raw.githubusercontent.com/jupyterlab/jupyterlab/main/docs/source/getting_started/changelog_assets/4.4-console.png" class="jp-screenshot">
```

Comment:
What do you think about moving it one paragraph up (to around line 24)? This way there is no screenshot between the list of settings and the paragraph below which says "Toggling all the...".

Reply:
Sounds good 👍